frontend/integration-tests/tests/event.scenario.ts

```diff
@@ -0,0 +1,56 @@
+import { browser, $, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+import * as crudView from '../views/crud.view';
+import * as yamlView from '../views/yaml.view';
+
+describe('Events', () => {
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  describe('Events', () => {
+    const name = `${testName}-pod`;
+    const testLabel = 'automatedTestName';
+
+    it('displays Events all resources', async () => {
+      await browser.get(`${appHost}/k8s/all-namespaces/events`);
+      await browser.wait(until.presenceOf($('.co-sysevent__box')));
+      expect($('.co-sysevent__box').isPresent()).toBe(true);
```

Comment:
This should never evaluate to false since we're waiting for it just above.

I'm not sure what value we get from this test, particularly because we check for a specific pod event below. I would probably just remove this in favor of the pod test.

Reply:
Since we added the route that test the 'all-namespaces" path, I am fine removing this.

Reply:
OK I see the difference now. I'm OK leaving it, although the wait-for and expect check on the same element is odd. It would probably be better to use the expect to check for something else.

Reply:
Let's leave it out for now. We can always come back and fill in these tests. Really we should be testing the filters, but I want to make sure the bug fix makes it in.